File: 110algoritmos/exercicios003/ex003-JavaScript/ex003.js
const readlineSync = require('readline-sync');

let n1 = Number(readlineSync.question('Digite o primeiro valor: '));
let n2 = Number(readlineSync.question("Digite o segundo valor: "));
const total = n1 + n2;
console.log(`A soma de ${n1} + ${n2} = ${total}`);

File: 110algoritmos/exercicios003/ex003-Csharp/ex003.cs
using System;

class Program {
    static void Main(){
        float num1, num2, total;

        Console.Write("Digite o primeiro valor: ");
        num1 = int.Parse(Console.ReadLine());
        Console.Write("Digite o segundo valor");
        num2 = int.Parse(Console.ReadLine());
        total = num1 + num2;
     //   Console.WriteLine("A soma de "+ num1 + " + " + num2 + " = " + total);
        Console.WriteLine($"A soma de {num1} + {num2} = {total}");
    }
}